thorlcr/activities/indexwrite/thindexwriteslave.cpp

```diff
@@ -243,6 +243,8 @@ class IndexWriteSlaveActivity : public ProcessSlaveActivity, public ILookAheadSt
             builder.clear();
             if (builderIFileIO)
             {
+                stat_type numDiskWrites = builderIFileIO->getStatistic(StNumDiskWrites);
+                stats.mergeStatistic(StNumDiskWrites, numDiskWrites);
```

Comment:
You haven't changed this to the more general:

            mergeStats(stats, builderIFileIO, diskWriteRemoteStatistics);

Which should publish more than one disk write statistic (e.g., time, total written) to the activity stats.

Reply:
was about to say same (re. general mergeStats method)

Reply:
> https://github.com/hpcc-systems/HPCC-Platform/pull/15030/files#r653428038

Did you want me to change something in this file?

Reply:
> You haven't changed this to the more general:
> 
> ```
>         mergeStats(stats, builderIFileIO, diskWriteRemoteStatistics);
> ```
> 
> Which should publish more than one disk write statistic (e.g., time, total written) to the activity stats.

@shamser - are you also going to publish some of the other stats. Gavin has suggested here?

Reply:
@jakesmith I wasn't going to as I didn't need the additional stats for the cost calculation.  However, I've have done that now.